appserver/tests/complex/postboot/testPostbootcommand.py

```diff
@@ -0,0 +1,38 @@
+#!/bin/python3
+import os
+import subprocess
+import time
+import urllib.request
+import testpayaralib
+
+# Setup
+testpayaralib.PAYARA_PATH_BIN = os.path.abspath("../../../../appserver/distributions/payara/target/stage/payara6/bin")
```

Comment:
What if I wanted to test this against web profile or a server not located in the default build dir (e.g. like in the Jenkins pipeline).
Could we add some means of reading in user input?

Reply:
I kept it simple for beginning, but you are right. Added.